src/init/consentless/init-fixed-slots.ts

```diff
@@ -2,12 +2,18 @@ import { getCurrentBreakpoint } from '../../lib/detect/detect-breakpoint';
 import { removeDisabledSlots } from '../consented/remove-slots';
 import { defineSlot } from './define-slot';
 
+const isTabletOrMobile = () => {
+	const breakpoint = getCurrentBreakpoint();
+	return breakpoint === 'mobile' || breakpoint === 'tablet';
+};
+
 const initFixedSlots = async (): Promise<void> => {
 	await removeDisabledSlots();
 
-	const isDCRMobile =
-		window.guardian.config.isDotcomRendering &&
-		getCurrentBreakpoint() === 'mobile';
+	// We remove top-above-nav on both tablet and mobile for consentless
+	// due to issues with ad sizing in OptOut at the tablet breakpoint
+	const shouldHideTopAboveNav =
```

Comment:
Don't you think `isDCRMobileOrTablet` might be a bit more obvious than `shouldHideTopAboveNav`?